### ISSUE TITLE
Merge multikey branch with trunk

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#677]
_[Trac time 20090605 170725Z]_ The multikey branch is currently undergoing final testing.  This ticket is a placeholder for that merge into the main trunk.

**Comment by robmcmullen** [comment]
_[Trac time 20090609 002200Z]_ 8a4ed3e03c6c297bb9a47e88c896e72ae15d36b4 Fixed #677: merged r1975:[old svn rev2115] from multikey branch onto mainline

**Comment by robmcmullen** [comment]
_[Trac time 20090605 194042Z]_ Looks like git-svn can't handle this, so have to operate on an svn repository:

```
svn co http://svn.flipturn.org/peppy/trunk peppy-svn
svn merge -r 1975:HEAD http://svn.flipturn.org/peppy/branches/multikey
```

and then `svn commit`
